src/components/InstancesList/InstancesList.js

```diff
@@ -1178,6 +1178,7 @@ class InstancesList extends React.Component {
             searchableIndexesPlaceholder={null}
             initialResultCount={INITIAL_RESULT_COUNT}
             initiallySelectedRecord={getItem(`${namespace}.${segment}.lastOpenRecord`)}
+            inputType="textarea"
```

Comment:
Textarea looks much higher to me
![image](https://github.com/folio-org/ui-inventory/assets/77053927/ffcbfac9-2aab-4c3e-b6fb-a47503e61c0a)

Reply:
Fixed
![image](https://github.com/folio-org/ui-inventory/assets/19309423/afaf9688-259b-4ed0-aa81-9bbe63695bfe)